Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:18.20-alpine AS build-stage
+
+RUN apk update  \
+  && apk add ruby
+
+COPY pix-site code/pix-site
+COPY shared code/shared
+
+ENV SITE=pix-site
+
+WORKDIR /code/shared
+RUN npm install
+
+WORKDIR ../pix-site
+RUN npm install
+RUN npm run build
+
+ENV PORT=80
+RUN erb servers.conf.erb > nginx.conf
```

Comment:
Le templating par defaut du buildpack nginx est bien erb, ce n'est cependant pas le cas de base pour Docker. 

Ici on se retrouve à installer ruby pour faire nos affaires, alors que la solution proposée par nginx est d'utiliser les templates (cf: https://hub.docker.com/_/nginx section "Using environment variables in nginx configuration (new in 1.19)")